### PR TITLE
add information about table init in readme and add script in prisma.j…

### PR DESCRIPTION
**Pull request by 01110111-wave**
…s to check whether the table exist

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor or other

# Checklist:

- [x] I have run `pnpm format` and my code don't have any linting issues
  
  
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by dtinth** [comment]
@01110111-wave 

you accidentally pushed `data/dock/db.sqlite`. can you delete it and add to `.gitignore`? thank you, sorry for trouble, it was late at night and i wasn't thinking clearly lol

**Comment by 01110111-wave** [comment]
Done, I really think that it might be mistake to publish the DB along with commit; however, as the precursor does, so do I.

File: src/prisma.ts [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
 
-export const prisma = new PrismaClient()
+const prisma = new PrismaClient()
+{
+  //This scope is only to check whether the table in DB exist.
+  //If table does not exist execute "pnpm prisma db push".
+  try {
+    await prisma.messageReportCount.findFirst()
+  } catch (error) {
+    console.log(error)
```
Comment (buskun) [inline src/prisma.ts:10]: Can we log the exact instruction in `README.md` in here?

Comment (01110111-wave) [reply src/prisma.ts:10]: Actually, the same instruction about running command 
```
pnpm prisma db push
```
is also in README.md

File: src/prisma.ts [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
 
-export const prisma = new PrismaClient()
+const prisma = new PrismaClient()
+{
+  //This scope is only to check whether the table in DB exist.
+  //If table does not exist execute "pnpm prisma db push".
+  try {
+    await prisma.messageReportCount.findFirst()
+  } catch (error) {
+    const errorTypeCasted: Error = error as Error
```
Comment (buskun) [inline src/prisma.ts:10]: Better to check the type `if (error instanceof Error)`

**Pull request closed without merging**